editor.planx.uk/src/@planx/components/Calculate/Public/index.tsx

```diff
@@ -24,5 +24,5 @@ export default function Component(props: Props) {
     });
   }, []);
 
-  return <p>Calculating…</p>;
+  return null;
```

Comment:
`null` is now more consistent with other "hidden" components like `SetValue`; rendering an un-Card-formatted "Calculating..." was a bit confusing in Storybook etc